File: test/Data.Analyzers.Vsix.Test/CSharp/DuplicateModelDeclaration.cs
using DevZest.Data.Annotations;

namespace DevZest.Data.Analyzers.Vsix.Test.CSharp
{
    [CheckConstraint(nameof(CK_AlwaysTrue), "CK")]
    [CheckConstraint(nameof(CK_AlwaysTrue), "CK")]
    public class DuplicateModelDeclaration : Model
    {
        [_CheckConstraint]
        private _Boolean CK_AlwaysTrue
        {
            get { return _Boolean.Const(true); }
        }
    }
}

File: test/Data.Analyzers.Vsix.Test/VisualBasic/DuplicateModelDeclaration.vb
Imports DevZest.Data.Annotations

<CheckConstraint(DuplicateModelDeclaration._CK_AlwaysTrue, "CK")>
<CheckConstraint(DuplicateModelDeclaration._CK_AlwaysTrue, "CK")>
Public Class DuplicateModelDeclaration
    Inherits Model

    Friend Const _CK_AlwaysTrue = NameOf(CK_AlwaysTrue)
    <_CheckConstraint>
    Private ReadOnly Property CK_AlwaysTrue As _Boolean
        Get
            Return _Boolean.Const(True)
        End Get
    End Property
End Class
